### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by FrederikBolding**
Bug fixes for bugs found by QA

### CODE REVIEW

File: src/features/SendAssets/SendAssets.reducer.tsx [hunk 1]
```diff
@@ -92,7 +92,7 @@ export const sendAssetsReducer = (state: State, action: ReducerAction): State =>
       const txReceipt = signedTx
         ? createPendingTxReceipt(state, action.payload.hash as ITxHash)
         : undefined;
-      return { ...state, txReceipt };
+      return { ...state, send: false, txReceipt };
```
Comment (blurpesec) [inline src/features/SendAssets/SendAssets.reducer.tsx:95]: can you explain what this does?

Comment (FrederikBolding) [reply src/features/SendAssets/SendAssets.reducer.tsx:95]: `send` marks that we are ready to send for signing for non web3 wallets. I just forgot to reset the value when the send was done.

File: src/services/ApiService/Gas/gasPriceFunctions.tsx [hunk 2]
```diff
@@ -50,6 +50,7 @@ export async function fetchGasPriceEstimates(network: Network): Promise<GasEstim
     const estimates: GasEstimates = await fetchGasEstimates();
     return estimates;
   } catch (err) {
+    console.error(err);
```
Comment (GuillaumeRx) [inline src/services/ApiService/Gas/gasPriceFunctions.tsx:53]: Do we want to leave that ?

Comment (FrederikBolding) [reply src/services/ApiService/Gas/gasPriceFunctions.tsx:53]: For next time when everything breaks.

Comment (GuillaumeRx) [reply src/services/ApiService/Gas/gasPriceFunctions.tsx:53]: Okay ! :)

**Pull request merged**